internal/cmd/beta/alb/quotas/quotas.go

```diff
@@ -0,0 +1,132 @@
+package quotas
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/goccy/go-yaml"
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/alb/client"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/tables"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/alb"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "quotas",
+		Short: "Gets the quotas",
+		Long:  "Gets the current quotes for the application loadbalancers.",
```

Comment:
```suggestion
		Short: "Shows the application load balancer quotas",
		Long:  "Shows the current quotes for the application load balancers.",
```
Only a nitpick, but stays consistent with the description of other quotas commands like the normal load balancer. I would also seperate load balancer into two words. https://github.com/stackitcloud/stackit-cli/blob/8ebf46681a9d460da851a66a7a75e9e5fdad4333/internal/cmd/load-balancer/quota/quota.go#L26-L30